src/xrEngine/device.cpp

```diff
@@ -34,6 +34,7 @@ ENGINE_API BOOL g_bRendering = FALSE;
 
 BOOL g_bLoaded = FALSE;
 ref_light precache_light = 0;
+int g_dwFPSlimit = 120;
```

Comment:
FPS lock is what GSC made in their multi-patch and this is the easiest decision.
Many users complained GSC that their video card whistled and uttered other sounds and eventually broke. They (GSC) posted about this somewhere, but I can't find where.